src/rgw/rgw_common.h

```diff
@@ -1632,6 +1643,10 @@ struct req_state : DoutPrefixProvider {
   int bucket_instance_shard_id{-1};
   string redirect_zone_endpoint;
 
+  std::stack<Span> stack_span;
```

Comment:
why not under ifdef?

Reply:
@yuvalif yes it should be inside under `#ifdef`, my bad i will update it soon